gammapy/scripts/analysis.py

```diff
@@ -0,0 +1,134 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Session class driving the high-level interface API"""
+import copy
+import jsonschema
+import logging
+from ..utils.scripts import read_yaml
+from astropy import units as u
+from pathlib import Path
+
+__all__ = ["Analysis"]
```

Comment:
Suggest to add `Config` here and have it show up in the docs. It will grow functionality and users will interact with it, so I think it's appropriate to have it show up in the API docs.